config/locales/fr.yml

```diff
@@ -247,7 +247,7 @@ fr:
       delivery_title:
         minister: Ministre
         speaker: Orateur
-      written_on: 'Rédiger le: '
+      written_on: 'Rédiger le :'
```

Comment:
This seems weird, is it a special case?

Reply:
Oh wow, I've learned something new today! :D

Reply:
+1!